File: 3888/3888.c
int minimumOperationsToMakeAllGrid(struct TreeNode* root) {
    if (!root) return 0;
    int l = minimumOperationsToMakeAllGrid(root->left);
    int r = minimumOperationsToMakeAllGrid(root->right);
    return 1 + (l > r ? l : r);
}

File: 3888/3888.cs
public class Solution {
    public int MinimumOperationsToMakeAllGrid(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumOperationsToMakeAllGrid(root.left), MinimumOperationsToMakeAllGrid(root.right));
    }
}